requirements.in

```diff
@@ -0,0 +1,8 @@
+pyperclip
```

Comment:
using pip-tools for every project now

Reply:
I'd love to know why.

Reply:
Easy to use and the requirements.txt it generates has more info how dependencies relate to each other (similar to pipdeptree).

Reply:
I'll see how that compares to poetry. Thanks.